src/System.Diagnostics.DiagnosticSource/pkg/System.Diagnostics.DiagnosticSource.pkgproj

```diff
@@ -4,8 +4,6 @@
   <PropertyGroup>
     <!-- we need to be supported on pre-nuget-3 platforms (Dev12, Dev11, etc) -->
     <MinClientVersion>2.8.6</MinClientVersion>
-    <!-- workaround for https://github.com/dotnet/buildtools/issues/1529 -->
-    <HarvestVersion>4.3.0</HarvestVersion>
```

Comment:
I just pushed an update to add the 4.4.0 package restore in harvesting and updated the packageindex accordingly.